src/main/kotlin/org/jitsi/nlj/util/PacketCache.kt

```diff
@@ -35,11 +35,36 @@ class PacketCache(
      */
     val size: Int = 500
 ) : NodeStatsProducer {
-    private val packetCaches: MutableMap<Long, RtpPacketCache> = ConcurrentHashMap()
+    private val packetCaches = Collections.synchronizedMap(
+        LinkedHashMap<Long, RtpPacketCache>(
+            // These are the default values of initialCapacity and loadFactor - we have to set them to be able to set
+            // accessOrder
+            16, /* initialCapacity */
+            0.75F, /* loadFactor */
+            true /* accessOrder */
```

Comment:
Why do we need this change (using `synchronizedMap`, and setting `accessOrder`)?

Reply:
To make the iteration order of the map oldest-first, so `expireCaches` doesn't have to walk the whole map, but rather can just query the front of the list until it finds something it doesn't need to expire.